src/tailwind.preset.js

```diff
@@ -18,8 +18,13 @@ export default /** @type {Partial<import('tailwindcss').Config>} */ ({
         DEFAULT: '#dbdbdb',
       },
       borderRadius: {
-        // Equivalent to tailwind default `rounded-sm` size
-        DEFAULT: '0.125rem',
+        // Equivalent to tailwind default `rounded-sm` size.
+        // These should be overwritten by downstream projects in order to
+        // "enable" rounded corners.
+        // Next major version will remove this config, effectively enabling
+        // rounded corners.
+        lg: 'var(--h-border-radius-lg, 0.125rem)',
+        DEFAULT: 'var(--h-border-radius, 0.125rem)',
```

Comment:
Downstream projects should not see any changes when updating the library, but they can overwrite the rounded border values via `--h-border-radius` and `--h-border-radius-lg` vars.

We will need to find a way to define CSS vars based on feature flags.